typescript/cli/anvil-start.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+# NOTE: This script is intended to be run from the root of the repo
+
+echo -e "\nYou must have 'anvil' from Foundry installed to proceed."
+echo -e '\nTo install, run the following command:'
+echo -e '\tcurl -L https://foundry.paradigm.xyz | bash && foundryup && anvil\n'
```

Comment:
Lets point people to foundry's docs instead of specifying instructions here. The steps may change or vary based on platform.